2_adventures/001/skahlert/04_day4_challenge-countdown/README.md

```diff
@@ -0,0 +1,11 @@
+#Day 4
+
+Here's a simple one:
+
+##Countown to the next deadline?
+
+Due to the time difference I have a tough time remembering what *Day* of the challenge we're in and when I have to deliver the pull request. So I made a little script to tell me exactly that.
```

Comment:
Wish I had seen this a few minutes ago... someone had said they were having a rough time (snicker... time... I said time...) and that there should be a tool for that!  I could have directed them here..